database/postgres/composition.yaml

```diff
@@ -17,7 +17,7 @@ spec:
           forProvider:
             region: us-west-2
             description: An excellent formation of subnetworks.
-          reclaimPolicy: Delete
+          deletionPolicy: Delete
```

Comment:
we can probably just delete these completely, no other resources in this repo are setting them

Reply:
@jbw976  happy either way but IMO kind of nice to have an example of the usage in one of the resources